requirements_test.txt

```diff
@@ -15,6 +15,7 @@ mock-open==1.4.0
 mypy==0.991
 pre-commit==2.21.0
 pylint==2.15.10
+pylint-per-file-ignores==1.1.0
```

Comment:
Together with the changes to `pyproject.toml` this is the only real change.
Everything under `tests` folder is just removing the now unnecessary pylint ignores.